samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/MainActivity.kt

```diff
@@ -204,6 +204,7 @@ open class MainActivity : AppCompatActivity(), LoginFragment.OnLoginCompleteList
         txtView.text = getString(R.string.signed_in, "${profile.displayName ?: ""} ${profile.email}")
     }
 
+    @Suppress("MissingSuperCall")
```

Comment:
Let's just add the call to `super` in an `else` block instead of suppressing :)

Reply:
Agreed.  Thanks